content/docs/reference/config/policy-and-telemetry/attribute-vocabulary.md

```diff
@@ -18,61 +18,95 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
+| `api.service` | string | The public service name. This is different than the in-mesh service identity and reflects the name of the service exposed to the client. | my-svc.com |
+| `api.version` | string | The API version. | v1alpha1 |
+| `api.operation` | string | Unique string used to identify the operation. The id is unique among all operations described in a specific &lt;service, version&gt;. | getPetsById |
+| `api.protocol` | string | The protocol type of the API call. Mainly for monitoring/analytics. Note that this is the frontend protocol exposed to the client, not the protocol implemented by the backend service. | "http", “https”, or "grpc" |
+| `request.auth.principal` | string | The authenticated principal of the request. This is a string of the issuer (`iss`) and subject (`sub`) claims within a JWT concatenated with “/” with a percent-encoded subject value. This attribute may come from the peer or the origin in the Istio authentication policy, depending on the binding rule defined in the Istio authentication policy. | accounts.my-svc.com/104958560606 |
+| `request.auth.audiences` | string | The intended audience(s) for this authentication information. This should reflect the audience (`aud`) claim within a JWT. | ['my-svc.com', 'scopes/read'] |
+| `request.auth.presenter` | string | The authorized presenter of the credential. This value should reflect the optional Authorized Presenter (`azp`) claim within a JWT or the OAuth2 client id. | 123456789012.my-svc.com |
+| `request.auth.claims` | map[string, string] | all raw string claims from the `origin` JWT | `iss`: `issuer@foo.com`, `sub`: `sub@foo.com`, `aud`: `aud1` |
+| `request.api_key` | string | The API key used for the request. | abcde12345 |
+| `check.error_code` | int64 | The error [code](https://github.com/google/protobuf/blob/master/src/google/protobuf/stubs/status.h#L44) for Mixer Check call. | 5 |
+| `check.error_message` | string | The error message for Mixer Check call. | Could not find the resource |
 | `check.cache_hit` | boolean | Indicates whether Mixer check call hits local cache. | |
 | `quota.cache_hit` | boolean | Indicates whether Mixer quota call hits local cache. | |
+
+## Deprecated attributes
+
+The following attributes have been renamed. We strongly encourage to use the replacement attributes:
+
+| Name | Replacement |
+|------|-------------|
+|`source.user`          |`source.principal`|
+|`destination.user`     |`destination.principal`|
+|`destination.service`  |`destination.service.host`|
+|`destination.name`     |`destination.service.name`|
+|`destination.namespace`|`destination.service.namespace`|
+
+Attribute `source.name` has been re-purposed to refer to the source workload instance name instead of the source service name.
+
+The following attributes have been deprecated and will be removed in subsequent releases:
+
+| Name | Type | Description | Kubernetes Example |
```

Comment:
The original names of the renamed attributes above will also be removed in a subsequent release.